pom.xml

```diff
@@ -166,6 +166,12 @@
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>rds</artifactId>
         </dependency>
+		<dependency>
+		    <groupId>com.amazonaws</groupId>
+		    <artifactId>aws-java-sdk-cognitoidp</artifactId>
+		    <version>1.11.771</version>
+		</dependency>
+		        
```

Comment:
Is this dependency needed? I was under the impression that it was only used for Identity Pool stuff?